### DISCUSSION TITLE
Ensuring Consistent Citekeys when Transitioning from BibTeX to Biblatex using BBT.

### DISCUSSION THREAD

**Discussion by hongyi-zhao** [retorquere/zotero-better-bibtex#2760] [General]
Dear Better BibTeX Author,

I am writing to seek your advice regarding a matter I've encountered during the preparation of my academic paper. I have been using Better BibTeX (BBT) for citation management, and it has been an immensely useful tool in my workflow.

Recently, I decided to make the transition from BibTeX to Biblatex for my document citations. I have already exported a BibTeX file using BBT and have used some of the citation entries in my paper. I wish to convert my current BibTeX bibliography database to the corresponding Biblatex format. However, a crucial concern for me is to maintain the same citekeys that I have already used in my LaTeX document so as to avoid the tedious task of updating citekeys manually in the TeX source file.

Could you please advise me on whether there is a systematic way to guarantee that the citekeys remain unchanged when converting from BibTeX to Biblatex within the BBT framework? Is there a specific procedure I should follow to ensure a seamless citation key transition?

I would greatly appreciate it if you could provide me with the guidance or instructions to accomplish this task efficiently. My goal is to facilitate a smooth transition without the need to adjust the previously cited references in my LaTeX source.

Thank you in advance for your attention to this matter. I am looking forward to your response.

Best regards, Zhao

**Comment by retorquere** [comment]
You will automatically get the same citation keys. You can just export using Better BibLaTeX and you will get biblatex output with the keys you already have.

**Reply by hongyi-zhao** [reply]
Got it. Thank you very much.
